backend/inspirehep/alembic/3637cb5551a8_update_values_in_authorschematype.py

```diff
@@ -42,10 +42,44 @@
 
 def upgrade():
     """Upgrade database."""
-    op.execute(f"DELETE FROM records_authors WHERE id_type NOT IN {new_enum_types}")
-    op.execute("ALTER TABLE records_authors ALTER COLUMN id_type TYPE VARCHAR(255);")
+    op.execute(
+        "CREATE TABLE tmp_records_authors (LIKE records_authors INCLUDING CONSTRAINTS);"
+    )
+    op.execute(
+        f"INSERT INTO tmp_records_authors SELECT * FROM records_authors WHERE id_type IN {new_enum_types};"
+    )
+    op.execute(
+        "ALTER TABLE tmp_records_authors ALTER COLUMN id_type TYPE VARCHAR(255);"
+    )
+    op.execute("DROP TABLE records_authors;")
+    op.create_index(
+        "ix_authors_records_author_id_id_type_record_id",
+        "tmp_records_authors",
+        ["author_id", "id_type", "record_id"],
+        unique=False,
+    )
+    op.create_index(
+        "ix_authors_records_record_id",
+        "tmp_records_authors",
+        ["record_id"],
+        unique=False,
+    )
+    op.create_index(
+        "ix_records_authors_id_type_authors_id",
+        "tmp_records_authors",
+        ["id_type", "author_id"],
+        unique=False,
```

Comment:
shouldn't this be unique?

Reply:
this is copy-paste from the previous migrations, actually from my understanding it shouldn't be unique cause you can have same author with same kind of id (recid) in different records